openfl/component/director/director.py

```diff
@@ -192,12 +198,63 @@ def get_envoys(self) -> list:
         logger.info(f'Shard registry: {self._shard_registry}')
         for envoy_info in self._shard_registry.values():
             envoy_info['is_online'] = (
-                time.time() < envoy_info['last_updated']
-                + envoy_info['valid_duration']
+                time.time() < envoy_info['last_updated'] + envoy_info['valid_duration']
             )
+            envoy_name = envoy_info['shard_info']['node_info']['name']
+            envoy_info['experiment_name'] = self.col_exp[envoy_name]
 
         return self._shard_registry.values()
 
+    def get_experiments(self, caller: str) -> list:
```

Comment:
We have a naming collapse here. 
To the existing `get_experiment_data` method we add `get_experiments` and `get_experiment`.
Could we resolve methods naming and merge some methods if possible?

Reply:
+1